### PR TITLE
Add script.py for easy installation

### PR DESCRIPTION
**Pull request by tanmaysachan**
Single script to apply the new vimrc

### CODE REVIEW

**Review by divyam3897 — COMMENTED** [review]
Update the README with the run instructions using the script and write a good commit message

File: script.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+os.chdir("vimrc")
+cur_path = os.path.dirname(os.path.abspath(__file__))
+print(cur_path)
+os.chdir(os.path.expanduser('~'))
+
+with open(".vimrc", "a") as cur_file:
+    cur_file.write("\nsource " + cur_path + "/.vimrc")
+
+subprocess.call(["vim", "+PluginInstall", "+qall"])
+
```
Comment (divyam3897) [inline script.py]: Remove all these empty lines.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# Installing the vimrc with script:
+* Run the python3 script from the folder where you want the Vimrc to be, it should take care of the rest. Or you could install it manually.
+```
+python3 script.py
+```
+
```
Comment (divyam3897) [inline README.md]: Remove the empty lines

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Squash all the commits into one and remove the irrelevant changes

File: script.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import subprocess
+os.system("git clone https://github.com/divyam3897/vimrc.git")
+os.chdir("vimrc")
+cur_path = os.path.dirname(os.path.abspath(__file__))
+print(cur_path)
+os.chdir(os.path.expanduser('~'))
+with open(".vimrc", "a") as cur_file:
+    cur_file.write("\nsource " + cur_path + "/.vimrc")
+subprocess.call(["vim", "+PluginInstall", "+qall"])
+
```
Comment (divyam3897) [inline script.py]: Remove these empty lines please!!

**Review by divyam3897 — CHANGES_REQUESTED** [review]

**Comment by tanmaysachan** [comment]
That should be it?

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Write a good commit message please. Follow https://chris.beams.io/posts/git-commit/

**Comment by divyam3897** [comment]
It has been merged thanks

**Pull request merged**